applications/tasks/src/modules/kt_compaction_reporter.erl

```diff
@@ -0,0 +1,474 @@
+%% within the same tuple, e.g.: {disk_start, disk_data, total_dbs}
+%% @end
+%%------------------------------------------------------------------------------
+-spec start_sizes_and_total_dbs(kt_compactor:dbs_and_sizes()) -> {non_neg_integer()
+                                                                 ,non_neg_integer()
+                                                                 ,non_neg_integer()
+                                                                 }.
+start_sizes_and_total_dbs(DbsAndSizes) ->
+    lists:foldl(fun start_sizes_and_total_dbs_fold/2, {0, 0, 0}, DbsAndSizes).
+
+-spec start_sizes_and_total_dbs_fold(kt_compactor:db_and_sizes()
+                                    ,{non_neg_integer()
+                                     ,non_neg_integer()
+                                     ,non_neg_integer()
+                                     }
+                                    ) -> {non_neg_integer()
+                                         ,non_neg_integer()
+                                         ,non_neg_integer()
+                                         }.
+start_sizes_and_total_dbs_fold({_Db, {Disk, Data}}, {DiskAcc, DataAcc, Counter}) ->
+    {DiskAcc + Disk, DataAcc + Data, Counter + 1};
+start_sizes_and_total_dbs_fold({_Db, _}, {DiAcc, DaAcc, Counter}) ->
+    %% 'not_found' | 'undefined' sizes
+    {DiAcc, DaAcc, Counter + 1}.
+
+-spec stats_to_status_fold(kz_term:ne_binary(), compaction_stats(), [kz_term:proplist()]) ->
+                                  [kz_term:proplist()].
+stats_to_status_fold(_Key, Stats, Acc) ->
+    Keys = ['id', 'found', 'compacted', 'queued', 'skipped', 'current_db',
+            'recovered_disk', 'pid', 'node', 'started'],
+    [[{kz_term:to_binary(Key), kz_term:to_binary(maps:get(Key, Stats))}
+      || Key <- Keys
+     ] | Acc].
+
+%%------------------------------------------------------------------------------
+%% @doc Print automatic compaction history according to user's input.
+%% @end
+%%------------------------------------------------------------------------------
+-spec history(kz_time:year(), kz_time:month()) -> 'ok'.
+history(Year, Month) ->
+    {'ok', AccountId} = kapps_util:get_master_account_id(),
+    Opts = [{'year', Year}, {'month', Month}, 'include_docs'],
+    case kazoo_modb:get_results(AccountId, ?AUTO_COMPACTION_VIEW, Opts) of
+        {'ok', []} ->
+            io:format("Not history found for ~p-~p (year-month)~n", [Year, Month]);
+        {'ok', JObjs} ->
+            Header = ["id"
+                     ,"found"
+                     ,"compacted"
+                     ,"skipped"
+                     ,"recovered"
+                     ,"started_at"
+                     ,"finished_at"
+                     ,"exec_time"
+                     ],
+            %% Last # is because the count of | (pipe) chars within `FStr' string.
+            HLine = lists:flatten(lists:duplicate(21+6+9+7+15+19+19+12+9, "-")),
```

Comment:
This seems like a static figure, why not replace this with the actual string (similar to [here](https://github.com/2600hz/kazoo/blob/master/applications/ecallmgr/src/ecallmgr_maintenance.erl#L612)) ?